modules/virt-about-monitored-resources-openshift-virtualization.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assembly:
+//
+// * virt/logging_events_monitoring/virt-about-monitored-resources-openshift-virtualization.adoc
+//
+
+[id="virt-about-monitored-resources-openshift-virtualization_{context}"]
+= About monitored resources in {VirtProductName} Overview
```

Comment:
The name of the page as it appears at the very top is: "Virtualization Overview." Therefore, I've updated the content to refer to that page.

We don't want to use the word "dashboard" because "Virtualization Overview" and "Dashboards" are both separate features although they are somewhat related to each other. We want to maintain separation between the two pages to avoid any confusion. 

Regarding the renaming of the module - since this information introduces the user to each of the "Cards" in the "Virtualization Overview" page, my logic is to leave it as it is. Each card on that page provides information about a specific resource that is monitored (for example, a donut chart showing the types of templates that have been deployed to VMs).

Attaching screenshots of the two different features: "Dashboards" and "Virtualization Overview".

![Screenshot from 2022-01-12 11-38-06](https://user-images.githubusercontent.com/55500825/149197036-cbc6853f-37e2-47e5-bfc1-db517abe0e7f.png)

![Screenshot from 2022-01-12 13-05-13](https://user-images.githubusercontent.com/55500825/149196849-d55cc78e-df0c-4c38-8b12-cae566ef9499.png)